tests/app/shared/issue/assignee/assignee.component.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { MaterialModule } from '../../../../../src/app/shared/material.module';
+import { PermissionService } from '../../../../../src/app/core/services/permission.service';
+import { Team } from '../../../../../src/app/core/models/team.model';
+import { UserService } from '../../../../../src/app/core/services/user.service';
+import { Phase } from '../../../../../src/app/core/models/phase.model';
+
+describe('AssigneeComponent', () => {
+  let component: AssigneeComponent;
+  let debugElement: DebugElement;
+  let nativeElement: HTMLElement;
+  let fixture: ComponentFixture<AssigneeComponent>;
+  let dummyIssue: Issue;
+
+  const testStudent: User = {
+    loginId: 'testStudent',
+    role: UserRole.Student
+  };
+
+  const testStudent2: User = {
+    loginId: 'testStudent2',
+    role: UserRole.Student
+  };
+
+  const testStudent3: User = {
+    loginId: 'testStudent3',
+    role: UserRole.Student
+  };
+
+  const testStudent4: User = {
+    loginId: 'testStudent4',
+    role: UserRole.Student
+  };
+
+  const dummyTeam: Team = new Team({
+    id: 'F09-2',
+    teamMembers: [testStudent, testStudent2, testStudent3, testStudent4]
+  });
+
+  const userService: any = jasmine.createSpyObj('UserService', [], { currentUser: testStudent });
+  const phaseService: any = jasmine.createSpyObj('PhaseService', [], { currentPhase: Phase.phaseTeamResponse, userService: userService });
+  const issueService: any = jasmine.createSpyObj('IssueService', ['getLatestIssue', 'updateIssue']);
+  const permissionsService: any = jasmine.createSpyObj('PermissionService', ['isIssueLabelsEditable']);
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        AssigneeComponent
+      ],
+      providers: [
+        UserService, IssueService, ErrorHandlingService, PhaseService, PermissionService
+      ],
+      imports: [
+        FormsModule, MaterialModule, BrowserAnimationsModule
+      ]
+    })
+    .overrideProvider(UserService, { useValue: userService })
+    .overrideProvider(IssueService, { useValue: issueService })
+    .overrideProvider(PhaseService, { useValue: phaseService })
+    .overrideProvider(PermissionService, { useValue: permissionsService })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    permissionsService.isIssueLabelsEditable.and.callFake(() => true);
+    fixture = TestBed.createComponent(AssigneeComponent);
+    component = fixture.componentInstance;
+
+    dummyIssue =  Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+    component.team = dummyTeam;
+    component.issue = dummyIssue;
+    fixture.detectChanges();
+
+    debugElement = fixture.debugElement;
+    nativeElement = fixture.nativeElement;
+  });
+
+  it('should have a placeholder value of - given no assignees', () => {
+    const matPlaceholderValue: HTMLElement = debugElement.query(By.css('.mat-select-placeholder')).nativeElement;
```

Comment:
```suggestion
    const matPlaceholderValue: HTMLElement = debugElement.query(By.css('.p')).nativeElement;
```

It's better to target the paragraph tag, which has the same placeholder value `-`
The `mat-select`'s placeholder is being used currently in our view

Reply:
Yea! ><